source/store/AppContext.js

```diff
@@ -0,0 +1,37 @@
+import PropTypes from 'prop-types';
+import React, { useState } from 'react';
+import env from 'react-native-config';
+import StorageService, { APP_ENV_KEY } from '../services/StorageService';
+
+const AppContext = React.createContext();
+
+function AppProvider({ children }) {
+  const acceptedModes = ['development', 'production'];
+  const defaultMode = env.APP_ENV || acceptedModes[0];
```

Comment:
Maybe check if env.APP_ENV is one of the accepted modes?
```suggestion
  const defaultMode =
    env.APP_ENV && acceptedModes.includes(env.APP_ENV) ? env.APP_ENV : acceptedModes[0];
```
Can't trust developers to set variables correctly, after all.

Reply:
Nope we are not trustworthy. Fixed!